src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsExtensions.cs

```diff
@@ -200,7 +200,9 @@ public static IResourceBuilder<AzureEventHubsResource> RunAsEmulator(this IResou
             // an event hub namespace without an event hub? :)
             if (builder.Resource.Hubs is { Count: > 0 } && builder.Resource.Hubs[0] is { } hub)
             {
-                var healthCheckConnectionString = $"{connectionString};EntityPath={hub.Name};";
+                var healthCheckConnectionString = connectionString.Contains(";EntityPath=") ?
```

Comment:
Instead of Contains, we could parse it properly and build a new one.

Reply:
Done. FYI, I can parse to test for the entitypath, but there is nothing in dotnet to build connection strings.

Reply:
@oising we already do this in the code, example:

```c#
var props = EventHubsConnectionStringProperties.Parse(connectionString);
if (string.IsNullOrWhiteSpace(props.EventHubName)) ...
```